extensions/liquibase/deployment/src/test/java/io/quarkus/liquibase/test/LiquibaseExtensionConfigFixture.java

```diff
@@ -0,0 +1,156 @@
+package io.quarkus.liquibase.test;
+
+import static org.junit.jupiter.api.Assertions.assertEquals;
+
+import java.util.function.Consumer;
+
+import javax.enterprise.context.ApplicationScoped;
+import javax.inject.Inject;
+
+import org.eclipse.microprofile.config.Config;
+import org.junit.jupiter.api.Disabled;
+
+import io.quarkus.liquibase.runtime.LiquibaseConfig;
+import io.quarkus.liquibase.runtime.LiquibaseDataSourceBuildTimeConfig;
+import liquibase.configuration.GlobalConfiguration;
+import liquibase.configuration.LiquibaseConfiguration;
+
+/**
+ * This fixture provides access to read the expected and the actual configuration of liquibase.
+ * It also provides a method combining all assertions to be reused for multiple tests.
+ */
+@Disabled
```

Comment:
Why this class needs a `Disabled` annotation?

Reply:
It's copy & paste from the Flyway extension. My understanding is that we do not use this class as a test class, but only as a test help class.

Reply:
@gastaldi should I remove it?

Reply:
If it is not necessary I believe it's better to remove it :wink:

Reply:
Done